.github/workflows/test.yml

```diff
@@ -132,7 +132,7 @@ jobs:
         run: |
           # On macOS we need to install some dependencies for testing
           brew install make
-          sudo xcode-select --switch /Applications/Xcode_11.7.app/Contents/Developer
```

Comment:
Interesting that this didn't match the version we are using. Someone more versed in GHA could perhaps make this use the `xcode-toolset-version` instead.

Reply:
We're not passing `xcode-toolset-version` into test.yml. :( We could start doing that, possibly, but we have never sent platform-specific arguments to test.yml before. In any case we'd still have to repeat the `xcode-toolset-version` in main.yml, but it would at least keep the version info to main.yml, so I guess it would be an improvement.